pkg/services/vmoperator/vmopmachine_test.go

```diff
@@ -373,7 +386,7 @@ var _ = Describe("VirtualMachine tests", func() {
 			requeue, err = vmService.ReconcileNormal(ctx, supervisorMachineContext)
 			verifyOutput(supervisorMachineContext)
 
-			Expect(vmopVM.Spec.ReadinessProbe.TCPSocket.Port.IntValue()).To(Equal(defaultAPIBindPort))
+			Expect(vmopVM.Spec.ReadinessProbe.TCPSocket.Port.IntValue()).To(Equal(6443))
```

Comment:
q why are we dropping the const?

Reply:
This was needed as part of moving the tests under a separate `_test` package. Reverted it since the change is not relevant to the PR.